src/pages/postgraphile/background_tasks.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+path: /postgraphile/background-tasks/
+title: Background Tasks in PostGraphile 
+---
+
+In your GraphQL server typically most processing will be done in the "frontground", as part of the request-response cycle. HTTP is a blocking protocol: once a client sends a request, it will wait for a reponse. This blocking nature requires the server to be fast at serving responses, to avoid keeping the client waiting for ever.
+
+But sometimes it is impossible to serve the response fast, for example we might need to do some heavy task like creating PDFs, calling an external API that takes a long time to respond, big calculations, or doing flaky tasks that need retries like sending an email. Or you might want to run some code periodically, detached from a user action. This are all good cases for background processing: the code runs outside of main request-response cycle, storing the result somewhere (typically on the database) and (optionally) notifying the client once it has finished.
+
+Task queues are systems that enable background processing for an application. They enable a way to persist this background processing tasks, notify the workers and handle errors and re-tries.
+
+### Tasks queues and workers for Postgres
+
+There are several exisiting solutions that can integrate with a PostgreSQL database or expose a Node.js-friendly API.
+
+- Graphile Worker
+- GCP Cloud Tasks
+- Faktory
+- RabbitMQ
+- ???
+
+#### Choosing your Task Queue
+
+An important bit you will need to consider is how are tasks created in your queue.
+
+What else should we add here???
+
+The recommend approach is to use Graphile Worker or Graphile Woker + another task queue through task delegation.
```

Comment:
```suggestion
Since we generally recommend database-driven development, the recommend approach is to use Graphile Worker (or Graphile Worker + another task queue through task delegation) since you can add jobs to the queue both from inside PostgreSQL functions (and thus triggers) and from any other source.
```